app.R

```diff
@@ -368,309 +371,333 @@ server <- function(input, output) {
       escape = FALSE
     )
   })
-  
-  
-  
-  
+
+
+
+
   # Plot: Visitors across Educational Resources ----------------------------------------------------
   output$plot_visitor_website <- renderPlot({
-    itcr_course_data() %>% 
+    itcr_course_data() %>%
       # Filter out ITN Website since it is not an "Educational Resource"
-      filter(!(website %in% c("ITN Website", "widget", "DaSL Collection", 
+      filter(!(website %in% c("ITN Website", "widget", "DaSL Collection",
                               "proof", "metricminer.org", "OTTR website",
-                              "Developing_WDL_Workflows"))) %>% 
+                              "Developing_WDL_Workflows"))) %>%
       ggplot(aes(x = reorder(website, -totalUsers), y = totalUsers, fill = target_audience)) +
       geom_bar(stat = "identity") +
       geom_text(aes(label = totalUsers), hjust = -0.2) +
-      coord_flip() +
+      coord_flip(clip="off") +
```

Comment:
Do we need this or can we just change the limits of the axes instead?

I read this about this option: 
clip
Should drawing be clipped to the extent of the plot panel? A setting of "on" (the default) means yes, and a setting of "off" means no. In most cases, the default of "on" should not be changed, as setting clip = "off" can cause unexpected results. It allows drawing of data points anywhere on the plot, including in the plot margins. If limits are set via xlim and ylim and some data points fall outside those limits, then those data points may show up in places such as the axes, the legend, the plot title, or the plot margins.

Reply:
but perhaps you tested that

Reply:
@carriewright11 Thanks for pointing this out! The reason I added it was to make sure the text labels weren't cut off. I use it in conjunction with adjusting the plot margin. If there's another method you prefer, I can totally switch it out for that. I tested it in that I looked at the plots side by side with how it currently was and my local preview and verified that the plots looked the same except for the text labels being visible.,  but I don't have any guarantee that it would never be a problem. My use case for it (and used it pretty frequently) is just to make sure that labels of some sort aren't cut off.

Reply:
Per conversation in meeting today, I will attempt to move the text labels onto the bars and just change the text or background color of the text